packages/fmlib_parse/fmlib_parse.0.5.2/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+
+synopsis: "Parsing with combinators and indentation sensitivity"
+
+description: """
+
+A parsing library with combinators in the style of Haskell's parsec.
+
+The parsing combinators have support for indentation sensitivity. Therefore it
+is easy to parse yaml files and indentation sensitive languages like Haskell and
+Python.
+
+Futhermore it is easy to generate user friendly error messages.
+
+"""
+
+maintainer: "Helmut Brandl <helmut.brandl@gmx.net>"
+
+authors: [ "Helmut Brandl <helmut.brandl@gmx.net>" ]
+
+license: "BSD-3-Clause"
+homepage: "https://github.com/hbr/fmlib"
+dev-repo: "git+https://github.com/hbr/fmlib.git"
+bug-reports: "https://github.com/hbr/fmlib/issues"
+
+
+build: [
+    ["dune" "subst"] {dev}
+    ["dune" "build" "-p" name "-j" jobs "@install" "@doc" {with-doc}]
+]
+
+
+depends: [
+    "ocaml" {>= "4.08.0"}
+    "dune" {>= "1.10"}
+    "odoc" {with-doc}
+    "ppx_inline_test" {>= "v0.13.0"}
+    "fmlib_std"    {>= "0.5.0"}
+    "fmlib_pretty" {>= "0.5.0"}
```

Comment:
Any reason to not use `=version` here?

Reply:
What does `=version` mean exactly?

Reply:
it means those packages should stay in sync with one another (fmlib_parse.0.5.2 requires fmlib_std.0.5.2). You've used it on all your other packages

Reply:
Thanks for the explanation. Then I think `=version` is better. Is it necessary to change it for this release or is it sufficient if I change it for the next release?